lib/mongo/session.rb

```diff
@@ -119,6 +176,20 @@ def ended?
       @server_session.nil?
     end
 
+    # Add the autocommit field to a command document if applicable.
+    #
+    # @example
+    #   session.add_autocommit!(cmd)
+    #
+    # @return [ Hash, BSON::Document ] The command document.
+    #
+    # @since 2.6.0
+    def add_autocommit!(command)
```

Comment:
`command` is not documented. I will stop commenting on this but the entire diff should be reviewed for undocumented parameters.

Reply:
Also, I imagine many of the added methods are not meant to be called from outside the driver and should be marked with `@api private` to indicate that.

Reply:
For the `add_*` methods, I followed the convention of the existing methods (e.g. `add_id`), which didn't put API private (or document their parameters).

Reply:
After reading driver code for the last month I found that it was easy to get a general idea of what the method does but specific parameter requirements are sometimes difficult to figure out. Hence the value of documentation should be in providing these details. For example, it seems like this method would mutate passed command but one can't tell for sure from the docstring.

Reply:
Makes sense! Let's file a ticket to sweep through the public API and document all the parameters that are missing; we can do it after the release.